### PR TITLE
 Minor update to the new project section

### PR DESCRIPTION
**Pull request by anuragsoni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ experienced OCaml programmer to participate).
   Open an issue on the "manifesto" repository explaining what the
   project is, where it is currently hosted (if anywhere), and what
   efforts you have made to contact the existing maintainers to make
-  sure they do not intend to continue maintenance.
+  sure they do not intend to continue maintenance (This can be an issue
```
Comment (pmetzger) [inline README.md]: You probably want a period still after "maintenance", and I'd rephrase the rest to

`(For example, you could have posted an issue on the original project, or have sent e-mail to the maintainers.)`

Comment (anuragsoni) [reply README.md]: Thanks for the feedback! I've updated the pull request.
